packages/core/src/index.html

```diff
@@ -215,6 +215,10 @@
         </script>
         <!-- End URL Handler -->
 
+        <!-- Start Cookies script -->
+        <script src="https://static.deriv.com/scripts/cookie.js" async></script>
```

Comment:
have we check if device_data get the information from cookie?

Reply:
in this project we retrieve most of the cookies from url_params except for data_first_contact and signup_device. Please refer to: [device_data](https://github.com/binary-com/deriv-app/blob/d7bc8f61588adaa21eb50c590c545547926b034b/packages/core/src/Stores/Helpers/device.js)